src/Bookings.js

```diff
@@ -1,21 +1,140 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import BookingForm from "./BookingForm.js";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
+import SearchResults from "./SearchResults.jsx";
 // import FakeBookings from "./data/fakeBookings.json";
 
-const Bookings = () => {
+const Bookings = props => {
+  const [bookings, setBookings] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [handleError, setHandleError] = useState(null);
+
+  function handleAddPerson(newBooking) {
+    setBookings([...bookings, newBooking]);
+  }
+
+  // console.log(bookings);
+
   const search = searchVal => {
     console.info("TO DO!", searchVal);
+    const filteredValue = bookings.filter(value => {
```

Comment:
Instead of `value` here, it could be good to use `booking`, it helps with readability, `value` is quite unclear what it actually is, but we know it is a `booking` from our `bookings` array!